views/includes/event/navigation.pug

```diff
@@ -0,0 +1,26 @@
+include ../../mixins/svg-elements
```

Comment:
Similarly here, the maps join together on mobile making it look like one super map. It needs to be spread out more